src/lib/auth/hooks.ts

```
@@ -0,0 +1,45 @@
+
+// ! 회원가입 hook
+export const useRegister = () => {
+  const navigate = useNavigate();
+  const { mutate } = useMutation({
+    mutationFn: registerFn,
+    onSuccess: (data) => {
+      console.log(data);
+
+      navigate("/auth/login");
+    },
+    onError: (error: AxiosError<RegisterErrorResponse>) => {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)